### ISSUE TITLE
MultiSelect: Role must have accessible name (ARIA issue)

### ISSUE THREAD

**Issue by KubaGorka** [primefaces/primereact#8279]
### Describe the bug

The MultiSelect component fails accessibility requirements. The input element with role="combobox" does not have an accessible name.

This can be observed in the official documentation:
https://primereact.org/multiselect

Example node:
`<input type="text" role="combobox" aria-expanded="false" tabindex="0" data-pc-section="input" readonly="" value="">
`

### System Information

```Shell
primereact: 10.9.7
```

### Steps to reproduce the behavior

Go to https://primereact.org/multiselect/

Inspect the input element of the MultiSelect

Notice that the role="combobox" element has no accessible name

### Expected behavior

The element should have an accessible name, either via aria-label, aria-labelledby, or an associated <label>.
